dev/breeze/src/airflow_breeze/utils/packages.py

```diff
@@ -474,14 +474,37 @@ def get_package_extras(provider_id: str, version_suffix: str) -> dict[str, list[
 
     :param provider_id: id of the package
     """
+
     if provider_id == "providers":
         return {}
     if provider_id in get_removed_provider_ids():
         return {}
+
+    deps_list = get_install_requirements(provider_id=provider_id, version_suffix=version_suffix)[1]
+
+    for i in range(len(deps_list)):
```

Comment:
LGTM, should we handle case `!=` ?

Reply:
I think it would be better to use requirements parser here 

```
        from packaging.requirements import Requirement
        req = Requirement(requirement_string)
        package = req.name
```

Reply:
Right, that makes it cleaner.